### ISSUE TITLE
The pipx PATH warning is back

### ISSUE THREAD

**Issue by chriscroome** [ansible/ansible-lint#3919]
The [issue fixed last summer](https://github.com/ansible/ansible-lint/issues/3541) is back with [the latest version](https://github.com/ansible/ansible-lint/releases/tag/v6.22.1):

```bash
ansible-lint .
WARNING: PATH altered to include /home/chris/.local/bin :: This is usually a sign of broken local setup, which can cause unexpected behaviors.

Passed: 0 failure(s), 0 warning(s) on 29 files. Last profile that met the validation criteria was 'production'.
```

```bash
which ansible-lint
/home/chris/.local/bin/ansible-lint

ls -lah ~/.local/bin/ansible-lint 
lrwxrwxrwx 1 chris chris 54 Jul 14 15:54 /home/chris/.local/bin/ansible-lint -> /home/chris/.local/pipx/venvs/ansible/bin/ansible-lint

ansible-lint --version
WARNING: PATH altered to include /home/chris/.local/bin :: This is usually a sign of broken local setup, which can cause unexpected behaviors.
ansible-lint 6.22.1 using ansible-core:2.16.0 ansible-compat:4.1.10 ruamel-yaml:0.18.5 ruamel-yaml-clib:0.2.8
```

On Debian Bookworm `pipx` is the recommended way to install PyPi packages so it would be good if using it didn't trigger warnings.
